ios_config.sh

```diff
@@ -102,6 +102,14 @@ if [[ ${_SEARCH_RESULT} ]]; then
     _PLIST_ENTRY_TYPES+=("string")
     _PLIST_ENTRY_VALUES+=("$_IOS_APP_ID")
   fi
+
+    # config.user_tracking_usage_description
+  _USER_TRACKING_USAGE_DESCRIPTION=$(getJsonKeyValue "$_JSON_OUTPUT_RAW" "user_tracking_usage_description")
+  if [[ $_USER_TRACKING_USAGE_DESCRIPTION ]]; then
+    _PLIST_ENTRY_KEYS+=("NSUserTrackingUsageDescription")
+    _PLIST_ENTRY_TYPES+=("string")
+    _PLIST_ENTRY_VALUES+=("$_USER_TRACKING_USAGE_DESCRIPTION")
+  fi
```

Comment:
@Salakar did you have any luck with the cocoapods add-plist functionality? I wonder if the reason we weren't using it before was the conditional JSON stuff, but ruby could probably handle it if you had a good result and maybe we could remove all this ios_config.sh machinery